gemrb/GUIScripts/bg2/GUIJRNL.py

```diff
@@ -148,6 +148,9 @@ def UpdateLogWindow (JournalWindow):
 		Label.SetText (45202)
 		list.sort(key=SortByDate)
 
+	Color = "800000"
+	if Section == 4:
```

Comment:
It will be valuable to add a comment what is `4` magic constant here

Reply:
@Mingun is defined on the same file: https://github.com/gemrb/gemrb/blob/33b898f32d5957ddb044552159a176d3d0a55ba1/gemrb/GUIScripts/bg2/GUIJRNL.py#L71 
I think it will be redundant but I can put a comment.

Reply:
- Because there are more that 70 lines between that lines it is better to add a comment
- In the future code could be refactored and both lines may turn out to be in different files and that link would be lost

Maybe even better to add a named constant and use it

Reply:
@Mingun a few points.

- 70 lines is that not long.
- If the file is split on the future the constant should be on the file or in another place ? 
- Should I move all **Sections** to constants ? Section var is already a **Global** one.

Let me know which is the best way to go.

Regards

Reply:
Journals are different among all the games, so there's not much opportunity to share more code. A comment never hurt though.

Reply:
Added a comment, don't think is a great one. Really don't know how to make it better.

Reply:
Just
```python
if Section == 4: # Journal
```
would be enough, but that is also good

Reply:
In general, it's more important to explain the why, not the what, which is often easily read from the code. Here there's not much different. I'd go with "only the personal journal section used a different title color", so readers know other section types have been checked already and the code is complete.

Reply:
@lynxlynxlynx @Mingun I change the comment, I like to put it above not on the side because the if can longer over the time.

Reply:
Eh, that's full of spelling errors, let's just use mine.